### PR TITLE
alpha and format as @Inputs

### PR DESCRIPTION
**Pull request by motabass**

### CODE REVIEW

**Comment by tiaguinho** [comment]
@motabass can you open an issue explaining these changes?

File: src/lib/color-picker/color-picker-collection.service.ts [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
 export class MccColorPickerCollectionService {
-
   /**
    * Hold current selected color
    */
   private _selectedColor: BehaviorSubject<string> = new BehaviorSubject<string>('');
 
+  alpha: boolean;
+
+  format: ColorStringFormat;
```
Comment (tiaguinho) [inline src/lib/color-picker/color-picker-collection.service.ts:15]: add comment

File: src/lib/color-picker/color-picker.component.spec.ts [hunk 2]
```diff
@@ -224,57 +226,59 @@ describe('MccColorPickerComponent', () => {
...
-    comp.selected.subscribe(selected => {
-      expect(selected).toBe(colors[1]);
-    });
-
     comp.backdropClick();
-  }));
+  });
 
-  it('should cancel selected color', fakeAsync(() => {
+  it('should cancel selected color', done => {
```
Comment (tiaguinho) [inline src/lib/color-picker/color-picker.component.spec.ts:249]: (done: DoneFn)

**Comment by motabass** [comment]
#155

**Pull request merged**